langchain4j/src/main/java/dev/langchain4j/service/DefaultAiServices.java

```diff
@@ -380,16 +378,26 @@ public Object invoke(Method method, Object[] args, InvocationContext invocationC
                         return actualResponse;
                     }
 
-                    private Optional<InvocationParameters> findInvocationParams(Object[] args, Parameter[] params) {
+                    private ChatRequestParameters chatRequestParameters(Method method, Object[] args, ToolServiceContext toolServiceContext, ResponseFormat responseFormat) {
+                        ChatRequestParameters defaultParams = ChatRequestParameters.builder()
+                                .toolSpecifications(toolServiceContext.toolSpecifications())
+                                .responseFormat(responseFormat)
+                                .build();
+                        return findParamOfType(ChatRequestParameters.class, args, method.getParameters())
```

Comment:
```suggestion
                        return findArgumentOfType(ChatRequestParameters.class, args, method.getParameters())
```

nit: argument is an actual value of a parameter

Reply:
Fixed.